buf.yaml

```diff
@@ -4,3 +4,4 @@ version: v1beta1
 build:
   roots:
     - third_party/oppia-ml-proto-0.0.0
+    - third_party/oppia-proto-api-introduce-proto-api-v1
```

Comment:
Adding this comment as a reminder for myself to update this with the specific commit not the branch name once the proto gets updated.